src/user.js

```diff
@@ -0,0 +1,27 @@
+class User {
```

Comment:
For the classes, since we are passing in parameters we would need to set the key values to be equal to that parameter.value.   This same concept would apply for the other classes as well.